osquery/core/init.cpp

```diff
@@ -168,6 +168,7 @@ DECLARE_bool(config_check);
 DECLARE_bool(config_dump);
 DECLARE_bool(database_dump);
 DECLARE_string(database_path);
+DECLARE_string(getfile_path);
```

Comment:
I'm hesitant about using an explicit path here. Perhaps making this default opaque to people and just use a secured temp folder is better.